config.json

```diff
@@ -1,6 +1,6 @@
 {
     "metadata_url":"https://api.airtable.com/v2/meta/",
     "records_url":"https://api.airtable.com/v0/",
-    "token":"airtable-api-key",
-    "base_id": "base-id"
+    "token":"keyQwkepkcSrWl1du",
```

Comment:
Might be a bit of a security risk uploading API key to github. Maybe we put in a .gitignore for the config.json and have an example_config.json that's just a blank template that we upload?